_posts/2019-12-03-agu-2019-pyopensci-events.md

```diff
@@ -38,9 +38,7 @@ programming language!
 ### AGU Event Resources - Sign Up To Be A Reviewer & Townhall Presentation
 
 Here are a few resources shared at the town hall:
-1. The <a href="https://docs.google.com/presentation/d/14UN-AD8p2S8q_gUX1SwvSjT7SCVBGID9kQ74_ZexS9s/edit?usp=sharing" target="_blank">pyopensci ignite talk </a>from the open source ignite session
-2. <a href="https://forms.gle/wvwLaLQre58YLHpD6" target="_blank">Sign up to stay involved with pyopensci using this google form. </a>
-</div>
+1. The <a href="https://docs.google.com/presentation/d/1emWah0WC9Or5uSH5IYyf0FfiPinLgqCr/edit?usp=sharing&ouid=112367566823345023071&rtpof=true&sd=true" target="_blank">pyopensci ignite talk </a>from the open source ignite session
```

Comment:
Agu link was down so i removed it and added an updated link to my presentation back in 2019

Reply:
this is an older presentation but i moved it from my cu org to pyos. so relinked it with a better up to date link that should work.